packages/@aws-cdk/cloudformation-include/lib/file-utils.ts

```diff
@@ -0,0 +1,6 @@
+import * as fs from 'fs';
+
+export function readJsonSync(filePath: string): any {
```

Comment:
Can we use this instead - https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md ?

Reply:
That's what I used originally. However, @eladb wants this to be part of MonoCDK, and there is a very strong pull to reduce external dependencies to the absolute minimum in MonoCDK. So, I removed the dependency on `fs-extra`, and wrote this utility method.